src/main/java/com/celements/web/service/CelementsWebScriptService.java

```diff
@@ -1100,4 +1103,28 @@ public String getSupportLinkURL() {
     return url;
   }
 
+  public String encodeUrlToUtf8(String urlStr) {
```

Comment:
please move implementation to xxxService and only add a delegation in the ScriptService